GameServerLib/Logic/Game.cs

```diff
@@ -255,6 +257,15 @@ private bool HandleDisconnect(Peer peer)
                 {
                     PacketNotifier.NotifyUnitAnnounceEvent(UnitAnnounces.SummonerDisconnected, peerinfo.Champion);
                 }
+                _logger.LogCoreInfo("somebody disconnected");
```

Comment:
Might want to throw in more info in there(like player clientID/netID and their name/champion name).
Also might want to start with capital letter.